src/Form/AdminSettings.php

```diff
@@ -434,8 +434,8 @@ public function buildForm(array $form, FormStateInterface $form_state) {
         For example: <strong>score desc, fgs_createdDate_dt desc</strong> will sort first by score, then by ingest date.<br />
         <strong>Note:</strong> only single-valued fields are sortable. Sorting by fields other than "score" will affect relevance.
         <br />For more information, see "sort" as discussed in the "Common query parameters" section of the 
-        <a href="!url">Solr documentation</a>.', [
-          '!url' => 'http://archive.apache.org/dist/lucene/solr/ref-guide/',
+        <a href="%url">Solr documentation</a>.', [
```

Comment:
I believe this is the wrong place holder: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Component%21Render%21FormattableMarkup.php/function/FormattableMarkup%3A%3AplaceholderFormat/8.2.x